contracts/UniswapV3Staker.sol

```diff
@@ -293,6 +293,8 @@ contract UniswapV3Staker is
             SafeMath.add(rewards[incentive.rewardToken][msg.sender], reward)
         );
 
+        stakes[params.tokenId][incentiveId].exists = false;
```

Comment:
I've updated Stake.exists to false, will this suffice or do we also want to set the liquidity to zero. The other option is to delete the stake from the mapping as cleanup.

Reply:
thanks Femi!! something like this should do the trick: 
```
delete stakes[params.tokenId][incentiveId];
```

since Stake has other variables. Clearing up _all_ those variables should actually save gas. Once again you can experiment with the gas snapshots